<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QMainWindow" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* Background */
QWidget {
    background-color: #f3f0ea;   /* soft cream */
    font-family: 'Inter';
    color: #3f3d38;              /* soft dark gray */
}

/* Login Card (Optional, create QWidget and set objectName to LoginCard) */
#LoginCard {
    background-color: #ffffff;
    border-radius: 18px;
    border: 1px solid #e8e2d9;
    padding: 25px;
}

/* Input Fields */
QLineEdit {
    background-color: #ffffff;
    padding: 10px;
    border-radius: 10px;
    border: 1px solid #cfc9c2;   /* subtle warm gray */
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #9d8f7e;   /* earthy brown */
}

/* Login Button */
QPushButton {
    background-color: #a8b389;   /* muted green */
    padding: 12px;
    border-radius: 10px;
    color: #ffffff;
    font-weight: bold;
    font-size: 15px;
}

QPushButton:hover {
    background-color: #94a272;   /* darker natural green */
}

/* Sign Up hyperlink-like label */
QLabel#SignUpLabel {
    color: #7c6f63;
}

QLabel#SignUpLabel:hover {
    color: #948676;
    text-decoration: underline;
}
</string>
   </property>
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>65</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Calibri&quot;;
</string>
    </property>
    <property name="text">
     <string>username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSignup">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
